GLTFSerialization/GLTFSerialization/GLTFHelpers.cs

```diff
@@ -184,7 +184,7 @@ private static long LoadBufferView(AttributeAccessor attributeAccessor, out byte
 			{
 				using (var memoryStream = attributeAccessor.Stream as System.IO.MemoryStream)
 				{
-					bufferViewCache = memoryStream.GetBuffer();
+					bufferViewCache = memoryStream.ToArray();
```

Comment:
Sorry not sure what this is supposed to help with? Why to switch to ToArray instead of GetBuffer?

Reply:
@blgrossMS any idea for why the conditional compile for not .NETCORE? The code after it seems to be doing the same thing.